src/components/Image/ImageLoader.jsx

```diff
@@ -71,14 +76,23 @@ export default class ImageLoader extends Component {
     if (this.shouldLoadPrimary()) {
       this.createLoader(this.props.src)
     } else if (this.shouldLoadFallbackUrl()) {
-      getDownloadLink(this.props.photo)
+      this.getDownloadLink(this.props.photo)
         .then(url => this.setState(state => ({ ...state, fallbackUrl: url })))
         .catch(error => this.allIsLost(error))
     } else if (this.shouldLoadFallback()) {
       this.createLoader(this.state.fallbackUrl)
     }
   }
 
+  // for compatibility reasons, we try to use cozy-client but fallback on cozy-client-js
```

Comment:
I am not sure, your PR is about adding Cozy-client, so when this code will be executed you *will* have a cozy-client and no fallback needed. Am I missing something ?

Reply:
Ce composant est également utilisé côté Drive, d'où cette précaution.